### DISCUSSION TITLE
OSM editors madness

### DISCUSSION THREAD

**Discussion by gander** [organicmaps/organicmaps#9672] [Ideas]
It's no surprise to me that current OSM editors would like Organic Maps to become their favorite tool for updating maps. This is a bad idea. This is a very bad idea. Why?

> Organic Maps is [...] offline maps app for **travelers**, **tourists**, **hikers**, and **cyclists**.

WE, OSM editors, already have tools to improve maps, and Organic Maps is the perfect project to spread the idea of ​​OSM to regular users. Adding more gimmicks for editors will only scare them away. The ability to leave a note for moderators is already enough information that everyone can contribute. That's all. No bells and whistles. Get a grip before you ruin a good project.

**Comment by fitojb** [comment]
Fearmonger

**Comment by ZeljkoBG** [comment]
Full support!
Someone wrote that OsmAnd is not a GPS application but GPS tools. Don't go that way.
Why meaningless options and Organic does not have the basic functions of a GPS application. Like the Menu for multipoint route, Demonstracion, Alternatves Routes,...
